server/ServerFSM.cpp

```diff
@@ -285,6 +285,18 @@ void ServerFSM::HandleNonLobbyDisconnection(const Disconnection& d) {
                 m_server.DropPlayerEmpireLink(id);
             }
         }
+        // add "player left game" message
+        boost::posix_time::ptime timestamp = boost::posix_time::second_clock::universal_time();
+        std::string data = (boost::format("[[PLAYER_LEFT_GAME,%s]]") % player_connection->PlayerName()).str();
```

Comment:
`boost::format` isn't necessary here. It'd be fine to do just:
`std::string data = "[[PLAYER_LEFT_GAME," + player_connection->PlayerName()).str() + "]]";`

That said, it's probably slightly easier to read the current version, so OK.

Reply:
Fixed.